### DISCUSSION TITLE
SearXNG is removing the Oscar theme

### DISCUSSION THREAD

**Discussion by mrpaulblack** [searxng/searxng#1167] [Announcements]
After a lot of internal discussion we decided to drop the Oscar theme from SearXNG and did so in https://github.com/searxng/searxng/pull/1153. If you still want to use it you can check out https://github.com/searx/searx.
The main reason for dropping it, is that we are a small developer and maintainer team and we dont have the resources to maintain two themes at the same time. We decided at the end of last year to continue to maintain the simple theme instead of oscar and to eventually remove it, which happened today...

Here are some of the reasons for that decision:
* Oscar is build using bootstrap 3 and it would have required us to update it to bootstrap 4 and then bootstrap 5 to bring it up to date
* The simple theme does not depend on any toolkit like Oscar, so we dont need to make changes to the theme every time something updates upstream as much
* Having 2 themes would mean every change would need to be impl. and maintained in both themes
* Simple theme has in our opinion a better mobile UI then oscar
* The simple theme is much lighter in resources. After some optimizations it now only loads 7 resources and 114kb when visiting the landing page compared to oscar that loads 12 resources and 384kb of data (comparing uncompressed)
* Simple theme has better keyboard navigation

We did a lot of improvements to the simple theme over the past months already and it has much improved to the version from SearX. Be sure to give it a try.

To all SearXNG instance maintainers: Its important that you update your `settings.yml` file to use the simple theme as default, instead of the oscar theme, otherwise your instance will throw an error to your users!
You just need to make sure this is set in `settings.yml`:
https://github.com/searxng/searxng/blob/31521f3a465fa2beb7e3fd7ed47621262a28f554/searx/settings.yml#L82

At last a big thanks to everyone who contributed to SearXNG, without all the help from everyone this would not have been possible :tada:

**Comment by ISZO-89** [comment]
Thanks for the info, but now i hostet the searxng website for me, and have layout problems on my android phone. At the beginnig it worked greate, but now ist just works on desktop, on android i have a layout problem, where the edges are cuted. I have no clue how to fix this

**Reply by return42** [reply]
See https://github.com/searxng/searxng/issues/4187

**Reply by unixfox** [reply]
Why are you commenting on a 3 year old thread 😂😂😂
